### PR TITLE
Feature/mx-1347 link cruft template

### PR DESCRIPTION
**Pull request by cutoffthetop**
# Added
- pull request template
- sphinx documentation
- pre-commit yaml check
- CHANGELOG file
- cruft template link

# Changes
- harmonized boilerplate

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,3 +91,13 @@ Accepted data will be ingested and processed asynchronously.
 - build container with `make container`
 - run directly using docker `make run`
 - start with docker-compose `make start`
+
+## commands
```
Comment (rababerladuseladim) [inline README.md:95]: I'd really like to keep the usage before the development section. I'd first target the general audience (=users) and later the more specific (=developers).

Comment (cutoffthetop) [reply README.md:95]: i just think all of this is only useful for developers.

a real _user_ section would be additional. and would include steps to pull the container from ghcr and start the service (without building it locally first)

**Comment by cutoffthetop** [comment]
> One comment and one question: This is not up to date with the cookiecutter-template, is it?

now it is cd8918a

File: README.md (hunk 1 above)
Comment (rababerladuseladim) [reply README.md:95]: alright, I yield

**Review by rababerladuseladim — APPROVED** [review]

**Pull request merged**